GitExternal.cmake

```diff
@@ -115,7 +122,12 @@ function(GIT_EXTERNAL DIR REPO tag)
       RESULT_VARIABLE nok ERROR_VARIABLE error
       WORKING_DIRECTORY "${GIT_EXTERNAL_DIR}")
     if(nok)
-      message(FATAL_ERROR "${DIR} clone failed: ${error}\n")
+      if(GIT_EXTERNAL_LOCAL_OPTIONAL)
+        message(WARNING "${DIR} clone failed: ${error}\n")
```

Comment:
STATUS is enough imo.

(I hate WARNING in cmake since it makes ERROR hard to find)